### PR TITLE
Added Humanize Python Package

### PR DESCRIPTION
**Pull request by iomaganaris**
- Added Humanize package
- Chose 'develop' as preferred version, as the latest release (0.5.1) is a lot older that the latest commit in master branch

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-humanize/package.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyHumanize(PythonPackage):
+    """Python package that contains humanization utilities"""
+
+    homepage = "https://github.com/jmoiron/humanize"
+    url      = "git@github.com:jmoiron/humanize.git"
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/py-humanize/package.py]: url should be https

File: var/spack/repos/builtin/packages/py-humanize/package.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    version('develop', git=url, branch='master', preferred=True)
+    version('0.5.1', git=url, tag='0.5.1')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/py-humanize/package.py]: preferred version should be released one and not the branch which is constantly changing

**Review by pramodk — COMMENTED** [review]
Addressing in the next commit

**Review by pramodk — APPROVED** [review]

**Review by matz-e — DISMISSED** [review]
Preferred develop is a no go. Create a 2018-02-githash referring to a commit.

**Pull request merged**